Makefile

```
@@ -24,4 +24,15 @@ copy_course_file:
 		> internal/test_helpers/course_definition.yml
 
 update_tester_utils:
-	go get -u github.com/codecrafters-io/tester-utils
\ No newline at end of file
+	go get -u github.com/codecrafters-io/tester-utils
+
+test_bot: build
+	CODECRAFTERS_REPOSITORY_DIR=./internal/test_helpers/scenarios/test_bot \
+	CODECRAFTERS_TEST_CASES_JSON="[ \
+		{\"slug\":\"a01\",\"tester_log_prefix\":\"stage_1\",\"title\":\"Stage #1: Bind to port 8000\"}, \
+		{\"slug\":\"a02\",\"tester_log_prefix\":\"stage_2\",\"title\":\"Stage #2: Test endpoint returns 200\"}, \
+		{\"slug\":\"a03\",\"tester_log_prefix\":\"stage_3\",\"title\":\"Stage #3: Single valid move\"}, \
+		{\"slug\":\"a04\",\"tester_log_prefix\":\"stage_4\",\"title\":\"Stage #4: Bratko-Kopek tests\"}, \
```

```suggestion
		{\"slug\":\"a04\",\"tester_log_prefix\":\"stage_4\",\"title\":\"Stage #4: Bratko-Kopek test\"}, \
```

(match official name)